MyEtherWallet/MyEtherWallet
# devop/translations-index

### Devop

- [x] Add PR label
- [x] No hard-coded strings

### Test procedure
`npm run start:history`
navigate to both `/ru` and `/en` make sure it functions properly
then try
`npm start`
make sure this functions properly


## note
lang paths wont work on hash mode
